app/uk/gov/hmrc/vatsubscription/services/UpdateReturnPeriodService.scala

```diff
@@ -41,7 +41,10 @@ class UpdateReturnPeriodService @Inject()(updateVatSubscriptionConnector: Update
   def constructReturnPeriodUpdateModel(updatedReturnPeriod: ReturnPeriod, welshIndicator: Boolean)
                                       (implicit user: User[_]): UpdateVatSubscription = {
 
-    val agentOrCapacitor: Option[AgentOrCapacitor] = user.arn.map(AgentOrCapacitor(_, None))
+    val agentContactDetails: Option[ContactDetails] = if(updatedReturnPeriod.transactorOrCapacitorEmail.isDefined)
```

Comment:
Sorry to be picky but could you structure this `if` statement to be a bit more readable please